saidump/saidump.cpp

```diff
@@ -399,7 +416,154 @@ void dumpGraph(const TableDump& td)
     std::cout << "}" << std::endl;
 }
 
-int main(int argc, char ** argv)
+#define SWSS_LOG_ERROR_AND_STDERR(format, ...) { fprintf(stderr, format"\n", ##__VA_ARGS__); SWSS_LOG_ERROR(format, ##__VA_ARGS__); }
+
+/**
+ * @brief Preprocess the input json file to make sure it's a valid json file.
+ */
+static sai_status_t preProcessFile(std::string file_name)
```

Comment:
input param as const std::string& fileName

Reply:
Fix it.